docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
 
-| param | description                      | default value |
-|-------|----------------------------------|---------------|
-| limit | The number of records to request | 20            |
-| sort  | What field to sort by            | id            |
-| q     | String to search by              | --            |
+
+::: tip
+The API may not return any data for successful requests if the user doesn't have adequate read permission
+:::
+
+
+
+
+### Get Item
+Gets a single item from within a given collection
+
+```http
+GET /items/[collection-name]/[primary-key-value]
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Retrieved item<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist, or item doesn't exist within collection    |
+
+#### Examples
+
+* Return the project item with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/items/projects/1
+  ```
+
+
+
+
+### Get Items
+Gets items from within a given collection
+
+```http
+GET /items/[collection-name]/
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| limit         | 200       | The number of items to request                   |
+| offset        | 0         | How many items to skip before fetching results   |
+| sort          | id        | CSV of fields to sort by [Learn More](#sorting)  |
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| filter[field] |           | Filter items using operators [Learn More](#filtering)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+| q             |           | Search string [Learn More](#search-query)  |
+| id            |           | CSV of primary keys to fetch  |
```

Comment:
Is this an actual existing parameter? Is it called `id`? We use the term primary key everywhere now, so this feels very outdated. If you want to fetch items by primary key, use `/items/projects/<pk>,<pk>,<pk>` instead